lib/liquid.rb

```diff
@@ -57,6 +57,7 @@ module Liquid
 require 'liquid/parser_switching'
 require 'liquid/tag'
 require 'liquid/block'
+require 'liquid/block_body'
```

Comment:
Why is block_body.rb even in this branch?  Liquid::BlockBody isn't used by any of the code in this branch.